source/common/runtime/runtime_features.cc

```diff
@@ -94,6 +94,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.vhds_heartbeats",
     "envoy.reloadable_features.wasm_cluster_name_envoy_grpc",
     "envoy.reloadable_features.upstream_http2_flood_checks",
+    "envoy.reloadable_features.use_scheme_header",
```

Comment:
nit: not sure what would be better, but making this a bit more descriptive would be good.